src/robusta/core/sinks/telegram/telegram_sink.py

```diff
@@ -18,6 +20,23 @@
 VIDEO_ICON = "\U0001F3AC"
 
 
+class FindingStatus(Enum):
```

Comment:
This looks exactly like the FindingStatus in base.py
Is there a good reason to duplicate it?

Reply:
Sure, let me check.

Reply:
done